station/app/api/api_v1/endpoints/local_trains.py

```diff
@@ -204,26 +141,26 @@ def get_train_status(train_id: str, db: Session = Depends(dependencies.get_db)):
     return obj
 
 
-@router.get("/masterImages")
+@router.get("/masterImages", response_model=lt_schemas.MasterImagesList)
 def get_master_images():
     """
-
+    get all availabel master images
     @return:
     """
-    return harbor_client.get_master_images()
+    return {"images": harbor_client.get_master_images()}
 
 
-@router.get("/getAllLocalTrains")
+@router.get("/trains", response_model=lt_schemas.LocalTrainSchemaList)
```

Comment:
```python
@router.get("", response_model=List[lt_schemas.LocalTrain])
```
return list of objects instead of a specific list type?

Reply:
done